src/core/router/info.cc

```diff
@@ -664,23 +709,20 @@ const std::uint8_t* RouterInfo::LoadBuffer() {
   return m_Buffer.get();
 }
 
-void RouterInfo::CreateBuffer(const PrivateKeys& privateKeys) {
-  m_Timestamp = kovri::core::GetMillisecondsSinceEpoch();  // refresh timestamp
-  std::stringstream s;
-  std::uint8_t ident[1024];
-  auto ident_len = privateKeys.GetPublic().ToBuffer(ident, 1024);
-  s.write(reinterpret_cast<char *>(ident), ident_len);
-  WriteToStream(s);
-  m_BufferLen = s.str().size();
+void RouterInfo::CreateBuffer(const PrivateKeys& private_keys) {
```

Comment:
I chose not to run clang-format on this function because it would've done more than just newline brace and, from the [contributing guide](https://github.com/monero-project/kovri/blob/master/doc/CONTRIBUTING.md#sending-your-work):
>please try to not mix formatting fixes with non-formatting commits

I'm guilty of mixing elsewhere but that's because of the amount of re-write that was involved (in those places).

If this brace bugs you enough, that can be fixed. Otherwise, I had planned on doing a complete style check-in / running clang-format on both files once RI refactoring was done.